target/linux/ramips/image/mt7621.mk

```diff
@@ -1103,6 +1103,19 @@ define Device/ubnt_unifi-nanohd
 endef
 TARGET_DEVICES += ubnt_unifi-nanohd
 
+define Device/unielec_u7621-01-16m
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := UniElec
+  DEVICE_MODEL := U7621-01
+  DEVICE_VARIANT := 16M
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt76x2 kmod-usb-ledtrig-usbport \
+	kmod-usb3 -wpad-basic-wolfssl
```

Comment:
Why do you disable wpad, this has radios?!

Reply:
Ooops, yes thats a mistake, will be resolved